forge-game/src/main/java/forge/game/GameActionUtil.java

```diff
@@ -177,13 +183,7 @@ public static final List<SpellAbility> getAlternativeCosts(final SpellAbility sa
                         final String[] k = keyword.split(":");
                         final Cost disturbCost = new Cost(k[1], true);
 
-                        SpellAbility newSA;
-                        if (source.getAlternateState().getType().hasSubtype("Aura")) {
-                            newSA = source.getAlternateState().getFirstAbility().copyWithManaCostReplaced(activator,
-                                    disturbCost);
-                        } else {
-                            newSA = sa.copyWithManaCostReplaced(activator, disturbCost);
-                        }
+                        SpellAbility newSA = source.getAlternateState().getFirstAbility().copyWithManaCostReplaced(activator, disturbCost);
```

Comment:
This breaks normal disturb cast:

`EDT > java.lang.NullPointerException: Cannot invoke "forge.game.spellability.SpellAbility.copyWithManaCostReplaced(forge.game.player.Player, forge.game.cost.Cost)" because the return value of "forge.game.card.CardState.getFirstAbility()" is null
	at forge.game.GameActionUtil.getAlternativeCosts(GameActionUtil.java:186)
	at forge.game.card.Card.getAllPossibleAbilities(Card.java:6435)`

Reply:
ugh probably because it doesn't has manacost